### PR TITLE
JBR-7511 migrate build platforms to OL8 for the release 2024.3

### PR DESCRIPTION
**Pull request by vprovodin**

### CODE REVIEW

File: jb/project/docker/Dockerfile.oraclelinux_x86_64 [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# NOTE: This Dockerfile is meant to be used from the mkdocker_aarch64.sh script.
```
Comment (mkartashev) [inline jb/project/docker/Dockerfile.oraclelinux_x86_64:1]: `mkdocker_aarch64.sh` -> `mkdocker_x86_64.sh`

File: jb/project/docker/Dockerfile.ubuntu2004_x86_64 [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# NOTE: This Dockerfile is meant to be used from the mkdocker_aarch64.sh script.
```
Comment (mkartashev) [inline jb/project/docker/Dockerfile.ubuntu2004_x86_64:1]: `mkdocker_aarch64.sh` -> `mkdocker_x86_64.sh`

File: jb/project/docker/mkdocker_x86_64.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -euo pipefail
+set -x
+
+# This script creates a Docker image suitable for building AArch64 variant
```
Comment (mkartashev) [inline jb/project/docker/mkdocker_x86_64.sh:6]: `for building AArch64 variant` -> `for building the x64 variant`

File: jb/project/docker/mkdocker_x86_64.sh [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+# of the JetBrains Runtime "dev" version.
```
Comment (mkartashev) [inline jb/project/docker/mkdocker_x86_64.sh:7]: I think the part about `"dev" version` can be dropped.

File: jb/project/docker/mkdocker_x86_64.sh [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+else
+    echo "boot JDK \"$BOOT_JDK_LOCAL_FILE\" present, skipping download"
+fi
+
+# Verify that what we've downloaded can be trusted.
+sha256sum -c - <<EOF
+$BOOT_JDK_SHA *$BOOT_JDK_LOCAL_FILE
+EOF
+
+docker build -t jetbrains/runtime:jbr21env_oraclelinux8_aarch64 -f Dockerfile.oraclelinux_aarch64 .
```
Comment (mkartashev) [inline jb/project/docker/mkdocker_x86_64.sh:25]: It seems to be trying to build an aarch64 version instead of x64

**Review by mkartashev — CHANGES_REQUESTED** [review]

File: jb/project/docker/Dockerfile.ubuntu2004_aarch64 [hunk 6]
```diff
@@ -0,0 +1,48 @@
+# NOTE: This Dockerfile is meant to be used from the mkdocker_aarch64.sh script.
+
+# Pull a concrete version of Linux that does NOT recieve updates after it's
+# been created.  This is so that the image is as stable as possible to make
+# image creation reproducible.
+# NB: this also means there may be no security-related fixes there, need to
+# move the version to the next manually.
+
+# jetbrains/runtime:jbr17env_aarch64
```
Comment (mkartashev) [inline jb/project/docker/Dockerfile.ubuntu2004_aarch64:9]: This name of the image is out of date

File: jb/project/docker/Dockerfile.ubuntu2004_x86_64 [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+# Pull a concrete version of Linux that does NOT recieve updates after it's
+# been created.  This is so that the image is as stable as possible to make
+# image creation reproducible.
+# NB: this also means there may be no security-related fixes there, need to
+# move the version to the next manually.
+
+# jetbrains/runtime:jbr17env_x86_64
```
Comment (mkartashev) [inline jb/project/docker/Dockerfile.ubuntu2004_x86_64:9]: This name of the image is out of date

**Review by mkartashev — CHANGES_REQUESTED** [review]

**Review by mkartashev — APPROVED** [review]

**Review by ujpv — APPROVED** [review]

**Review by avu — APPROVED** [review]
LGTM

**Review by YaaZ — APPROVED** [review]

**Pull request merged**